pkg/envoy/ads/response.go

```diff
@@ -16,71 +20,91 @@ const (
 	ADSUpdateStr = "ADS"
 )
 
-// Wrapper to create and send a discovery response to an envoy server
-func (s *Server) sendTypeResponse(tURI envoy.TypeURI,
-	proxy *envoy.Proxy, server *xds_discovery.AggregatedDiscoveryService_StreamAggregatedResourcesServer,
-	req *xds_discovery.DiscoveryRequest, cfg configurator.Configurator) error {
+// getXDSResource function will invoke the right TypeURI handler for the resources associated with a proxy
```

Comment:
It would be great to make the rename/refactor in a separate - much smaller PR - so it is easier and faster to review.